src/test/ui/impl-trait/explicit-generic-args-with-impl-trait/const-args.rs

```diff
@@ -1,3 +1,8 @@
+// check-pass
+
+#![allow(incomplete_features)]
+#![feature(generic_const_exprs)]
```

Comment:
Why is this necessary? Running this on nightly with only `#![feature(explicit_generic_args_with_impl_trait)]` (which is what's being stabilized here) succeeds as expected.

Reply:
Huh, you're right I don't need these, I'm not sure why I thought I did, I think I copy-pasted them from a similar test without thinking too much.